install.rst

```diff
@@ -18,49 +18,136 @@
 Installing Zeek
 ===============
 
-Prerequisites
+To run Zeek, grab our official Docker images, download our Linux binary
+packages, install via Homebrew on your Mac, use the ports collection on FreeBSD,
+or build Zeek yourself.
+
+Docker Images
 =============
 
-Before installing Zeek, you'll need to ensure that some dependencies
-are in place.
+We provide `official Docker images <https://hub.docker.com/u/zeekurity>`_ on the
+Docker Hub. These images are based on Debian 11 and feature a complete Zeek
```

Comment:
That `11` will need updating often, would just mentioning Debian be enough?

Reply:
Done